src/Config/Converters/GraphQLRuntimeOptionsConverterFactory.cs

```diff
@@ -35,11 +55,79 @@ private class GraphQLRuntimeOptionsConverter : JsonConverter<GraphQLRuntimeOptio
                 return new GraphQLRuntimeOptions(Enabled: false);
             }
 
-            // Remove the converter so we don't recurse.
-            JsonSerializerOptions innerOptions = new(options);
-            _ = innerOptions.Converters.Remove(innerOptions.Converters.First(c => c is GraphQLRuntimeOptionsConverterFactory));
+            if (reader.TokenType == JsonTokenType.StartObject)
+            {
+                // Initialize with Nested Mutation operations disabled by default
+                GraphQLRuntimeOptions graphQLRuntimeOptions = new();
+                NestedMutationOptionsConverter nestedMutationOptionsConverter = options.GetConverter(typeof(NestedMutationOptions)) as NestedMutationOptionsConverter ??
+                                            throw new JsonException("Failed to get nested mutation options converter");
+
+                while (reader.Read())
+                {
+
+                    if (reader.TokenType == JsonTokenType.EndObject)
+                    {
+                        break;
+                    }
+
+                    string? propertyName = reader.GetString();
```

Comment:
What if propertyName is null here? Is it possible? If not, why is its type a nullable string?

Reply:
The type is kept nullable because of the return type of `reader.GetString()`. `propertyName` is not null when the flow reaches this point. A `null` property name is invalid and should get caught before the flow reaches here. 

However, I've added a null check, and an exception is thrown if `propertyName` turns out to be `null`.